Zigbee2MqttAssistant/Startup.cs

```diff
@@ -65,11 +71,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 				app.UseExceptionHandler("/Home/Error");
 			}
 
+			app.UseHttpsRedirection();
+
```

Comment:
You are forcing the app to work with `https`... which could break the HASS.IO integration and force the developer to create a self-sign certificate.

I think this should be configureable in the config.  What do you think?

Reply:
Making this selectable via configuration is a good idea.